core/src/main/scala/io/scalac/mesmer/core/actor/ActorCellDecorator.scala

```diff
@@ -1,15 +1,14 @@
 package io.scalac.mesmer.core.actor
 
-import io.scalac.mesmer.core.util.ReflectionFieldUtils
+import io.opentelemetry.instrumentation.api.field.VirtualField
 
 object ActorCellDecorator {
 
-  final val fieldName = "_actorCellMetrics"
+  @inline def getMetrics(actorCell: Object): Option[ActorCellMetrics] = Option(
```

Comment:
It was hard to find a least common type for `Cell` `ActorCell` and `ActorContext`. Before, we also operated on Object as an input parameter to `ActorCellDecorator.get()` method. 

@worekleszczy do you have any ideas on how to widen the type? If not, let's postpone this I guess. This will (should?) be simplified when we will attempt getting rid of the akka extension I guess.

Reply:
> This might lead to all Object having additional field if jvm even allows this

Could you post proof showing that/some more details of your investigation? I'd expect some significant performance/memory degradation if that would be the case, yet I didn't notice anything worrying.
For me now the part that is more convincing is just avoiding the Object type that could be anything. 

>  I'll take a stub at this and maybe will figure out sth.

I will do this. Above I was asking because you might have known from the top of your head.

Reply:
That should do the trick: 22d5da2641abf48fab14e4d135e72736ed00b223